cookbook/security/remember_me.rst

```diff
@@ -48,14 +43,61 @@ are shown here:
                 'main' => array(
                     'remember_me' => array(
                         'key'      => '%secret%',
-                        'lifetime' => 31536000, // 365 days in seconds
+                        'lifetime' => 604800, // 1 week in seconds
                         'path'     => '/',
-                        'domain'   => '', // Defaults to the current domain from $_SERVER
                     ),
                 ),
             ),
         ));
 
+The ``remember_me`` firewall defines the following configuration options:
+
+``key``
+    (default value: ``null``) The value used to encrypt the cookie's content.
+    It's common to use the ``secret`` value defined in the ``app/config/parameters.yml``
+    file.
+
+``name``
+    (default value: ``REMEMBERME``) The name of the cookie used to maintain the
+    user logged in. If you enable the "Remember Me" feature in several firewalls
+    of the same application, make sure to choose a different name for the cookie
+    of each firewall. Otherwise, you'll face lots of security related problems.
+
+``lifetime``
+    (default value: ``31536000``) The number of seconds during which the user
+    will remain logged in. By default users are logged in for one year.
+
+``path``
+    (default value: ``/``) The path where the cookie associated with this
+    feature is used. By default the cookie will be applied to the entire website
+    but you can restrict to a specific section (e.g. ``/forum``, ``/admin``).
+
+``domain``
+    (default value: ``null``) The domain where the cookie associated with this
+    feature is used. By default cookies use the current domain obtained from
+    ``$_SERVER``.
+
+``secure``
+    (default value: ``false``) If ``true``, the cookie associated with this
+    feature is sent to the user through an HTTPS secure connection.
+
+``httponly``
+    (default value: ``true``) If ``true``, the cookie associated with this
+    feature is sent to the user exclusively through an HTTP non-secure connection.
```

Comment:
This explanation looks wrong to me. ``httponly`` refers to the ``httponly`` flag of cookies, which are telling the browser that the cookie should be transmitted for HTTP requests but that it should not be exposed to client-side code.